### PR TITLE
feat(run, exec): Adds `--force-local` flag to exec and run commands

### PR DESCRIPTION
**Pull request by Noam3kCH**
When executing run and exec commands, execution order was not per the dependancy graph in the case where local dependencies had unmatched versions. This PR adds `--force-local` flag support for these command to allow running them in topological order.

## Motivation and Context
in our environment we use `lerna bootstrap --force-local` which works as expected, however when executing `lerna run build` the build order is not per dependency chain, causing unexpected errors. 

## How Has This Been Tested?
We added a test, in coordination with the `bootstrap --force-local` test to check the specific case.
namely, mismatching package dependencies and expecting them to run in the correct topological order

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by amiryonatan** [comment]
hey @evocateur,
We added support for --force-local in a few more commands. 
Can you please give us your feedback ? 

Thanks!

File: utils/listable/lib/listable-format.js [hunk 1]
```diff
@@ -37,12 +37,13 @@ function parseViewOptions(options) {
     showNDJSON: options.ndjson,
     showParseable: options.parseable,
     isTopological: options.toposort,
+    forceLocal: options.forceLocal,
```
Comment (evocateur) [inline utils/listable/lib/listable-format.js]: I'm not a fan of `--force-local` in the first place, and this isn't appropriate, as none of the listable commands accepts `--force-local` (nor should they).

File: commands/exec/README.md [hunk 2]
```diff
@@ -79,3 +79,11 @@ Pass `--no-bail` to disable this behavior, executing in _all_ packages regardles
...
 Disable package name prefixing when output is streaming (`--stream` _or_ `--parallel`).
 This option can be useful when piping results to other processes, such as editor plugins.
+
+### `--force-local`
+
+```sh
+$ lerna exec --force-local <command>
+```
+
+When passed, this flag causes the `exec` command to always execute in topological order regardless of matching version range.
```
Comment (evocateur) [inline commands/exec/README.md]: That's not an accurate statement. When your local dependent ranges are not satisfied, they are not considered local dependencies, thus they are not topologically ordered, as they have no topological relationship.

**Comment by Noam3kCH** [comment]
@evocateur
The motivation behind this PR came after we ran into unexpected behaviour where after you bootstrap with `--force-local` and have everything linked, the run command will execute in the same manner. 
This allows for example to test if some module dependencies which are behind could be updated to latest in single build command. Otherwise the use of the force local command is very limiting. 

Please let us know if you would like us to fix this pr, and in what approach, or drop it.
Thanks!

**Comment by amiryonatan** [comment]
@evocateur please advise on how to proceed?

**Comment by 3y3** [comment]
Is this pr not approved only for `README.md` reason?
@evocateur can you help to approve pr? Any idea how to fix readme?

**Pull request closed without merging**